vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -230,6 +233,16 @@ def parse_container_port_config(port_config, pod_id, container_name)
       }
     end
 
+    def parse_service_port_config(port_config, service_id)
+      {
+        :ems_ref     => "#{service_id}_#{port_config["port"]}_#{port_config["targetPort"]}_#{port_config["protocol"]}",
```

Comment:
@abonas yes but at least in containers you have some fields that can be updated (`restart_count`, `state`, etc.). Here it's pretty much nothing at this point.